admin/modules/config/settings.php

```diff
@@ -974,6 +974,12 @@
 			}
 		}
 
+		// Administrator is changing the login method.
+		if($mybb->settings['username_method'] != $mybb->input['upsetting']['username_method'] && $mybb->input['upsetting']['username_method'] == 1 || $mybb->input['upsetting']['username_method'] == 2)
+		{
+			$db->update_query('settings', array('value' => 0), "name='allowmultipleemails'");
```

Comment:
Not a big deal, but it could be moved here: https://github.com/Sama34/mybb/blob/fix-298/admin/modules/config/settings.php#L925 and the query changed to `$mybb->input['upsetting']['allowmultipleemails']` to possibly save a query.

Also, what if the user tries to change the multiple Emails setting after saving the username method? That wouldn't be blocked.